Xamarin.Forms.Platform.Android/ContextExtensions.cs

```diff
@@ -86,11 +86,7 @@ internal static double GetThemeAttributeDp(this Context self, int resource)
 		[MethodImpl(MethodImplOptions.AggressiveInlining)]
 		static void SetupMetrics(Context context)
 		{
-			if (s_displayDensity != float.MinValue)
-				return;
-
-			using (DisplayMetrics metrics = context.Resources.DisplayMetrics)
-				s_displayDensity = metrics.Density;
+			s_displayDensity = context.Resources.DisplayMetrics.Density;
```

Comment:
I don't think we need to remove the cached value for this one.

When you change the density the MainActivity gets recreated or if the user has specified ConfigurationChange.Density then it just runs the ConfigurationChanges method

So I think we can just reset the cache density from inside FormsAppCompatActivity in the OnCreate and OnConfiguraitonChanges methods

thoughts?

Reply:
cool, i will try it! 👍👍